.gitignore

```diff
@@ -63,16 +66,11 @@ extensions/core/
 experimental/
 
 # nupic bindings swig libraries
-nupic/bindings/*engine_internal.*
-nupic/bindings/*algorithms.*
-nupic/bindings/*iorange.*
-nupic/bindings/*math.*
-nupic/bindings/*.cxx
-nupic/bindings/proto/*.capnp
+nupic/bindings/
+~nupic/bindings/*.i
```

Comment:
`~` = `not`, so these lines mean ignoring all files in `nupic/bindings/`, except `*.i` files.